pkg/config/node.go

```diff
@@ -144,6 +152,41 @@ func getClusterConfigMapInstallConfig(configPath string) (installConfig types.In
 	return ic, err
 }
 
+func getBootstrapIP(apiVip string) (bootstrapIP string, err error) {
+	conn, err := net.Dial("tcp", net.JoinHostPort(apiVip, "64444"))
```

Comment:
Do we need o handle the add-master use case?
And I think it will safer to use here DialTimeout instead of Dial

Reply:
getBootstrapIP will just return empty string. net.Dial is blocking. If you mean that a new master comes online, as long as the master has a SVC entry, it should be picked up.

Reply:
My bad. I meant that yes, it blocks, but not if there is a connection refused, which is what will happen once the bootstrap goes down. Then it just returns immediately. DialTimeout probably should be used anyway.